doc/other/maxima/geod.mac

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
copyright info

Reply:
similarly add your name to the contributors in the begging of every file that you change

Reply:
@vissarion Regarding the copyright, one whose behalf should I add this? And since `geod.mac` file is taken from GeographicLib, shouldn't it contain its respective copyright?

Reply:
>Regarding the copyright, one whose behalf should I add this?

Just yourself. You can acknowledge gsoc. 

>And since geod.mac file is taken from GeographicLib, shouldn't it contain its respective copyright?

You can add something similar to this:
https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/srs/projections/proj/aea.hpp

Reply:
I've just added the [copyright information and updated the contributors list](https://github.com/boostorg/geometry/pull/486/commits/82c5c4d20c8f51dbd9d9e788b824ad06675a897e). Please let me know if it requires changes.

Reply:
"on behalf of" sounds a bit unclear. What about, "as part of"? Or "partially supported by gsoc18" in a separate sentence?

Reply:
I have replaced it with "... as part of Google Summer of Code 2018 program".